elephant/conversion.py

```diff
@@ -152,3 +155,704 @@ def binarize(spiketrain, sampling_rate=None, t_start=None, t_stop=None,
     else:
         return res, pq.Quantity(np.arange(t_start, t_stop+sampling_period,
                                           sampling_period), units=units)
+
+###############################################################################
+#
+# Methods to calculate parameters, t_start, t_stop, bin size, number of bins
+#
+###############################################################################
+def calc_tstart(num_bins, binsize, t_stop):
```

Comment:
These are essentially one-line code snippets, used in exactly one place each.  So I don't think they should be functions at all, and certainly not public API.